changelog/7955.deprecation.rst

```diff
@@ -0,0 +1 @@
+deprecated `ctype1="solar-x"/"solar_x"` and `ctype2="solar-y"/"solar_y"` in `sunpy.map.Mapbase.GenericMap.coordinate_system`
```

Comment:
We will need to workshop this message.

Reply:
> We will need to workshop this message.

I am sorry but I didn't get what you mean.